frontend/components/SetupPage/Create/SetupBridgeOnboarding/SetupBridgeOnboarding.tsx

```diff
@@ -20,6 +21,8 @@ export const SetupBridgeOnboarding = () => {
   const [quoteId, setQuoteId] = useState<Nullable<string>>(null);
   const [transferAndReceivingAmounts, setTransferAndReceivingAmounts] =
     useState<Nullable<CrossChainTransferDetails>>(null);
+  const [bridgeRetryOutcome, setBridgeRetryOutcome] =
```

Comment:
why outcome? but not status or result?

Reply:
I guess it's just my own thing, the fact that "outcome" is now associated with plans for future weeks :D result or status is more common, but fine with me